javaagent-exporters/logging/src/main/java/io/opentelemetry/javaagent/exporters/logging/LoggingExporter.java

```diff
@@ -33,7 +33,7 @@ public LoggingExporter(String prefix) {
   @Override
   public CompletableResultCode export(Collection<SpanData> list) {
     for (SpanData span : list) {
-      System.out.print(prefix + " " + span.getName() + " " + span.getSpanId() + " ");
+      System.out.printf("%s %s %s %s", prefix, span.getName(), span.getTraceId(), span.getSpanId());
```

Comment:
`printf` is sloooow - can we switch back to appending?

Reply:
Also looks like we're missing space between span ID and attributes now probably good to have that (or maybe enclose attributes in `{}`

Reply:
No objection to switching back to appending. Good catch on the missing space 👍

Reply:
@anuraaga Any numbers to back that claim? :) And it shit important enough for us?

Reply:
Have benchmarked similar now and again, generally 10-20x slower. The readability difference is small enough to get the free speed, and I always recommend people to just never use printf in Java, unless formatting numbers, since it's not worth the cognitive load of deciding between one or the other the performance difference is too great. So I'd consider it important for us.

Reply:
I will submit a PR to use the appending. 

Would it be better to use string builder